Examples/BookingSystem.AspNetFramework/Stores/SellerStore.cs

```diff
@@ -23,7 +23,7 @@ public AcmeSellerStore(bool useSingleSellerMode)
         }
 
         // If the Seller is not found, simply return null to generate the correct Open Booking error
-        protected override async ValueTask<ILegalEntity> GetSeller(SellerIdComponents sellerIdComponents)
```

Comment:
Is this a style thing or is there a functionality difference?

Reply:
It was just inconsistent with AspNetCore, so was updating it to match